resources/authgear/templates/en/web/authflowv2/wechat.html

```diff
@@ -17,13 +13,6 @@ <h1 class="screen-title">
     <h2 class="screen-description">
       {{ template "v2.page.wechat-auth.default.app-description" }}
     </h2>
-    {{ template "authflowv2/__alert_message.html"
-      (dict
-        "Type" "error"
-        "Classname" "mt-4"
-        "Message" $unknown_error_message
-      )
-    }}
```

Comment:
The baseline is every page should has `__alert_message.html` to show error. So the fix should be

- Keep `__alert_message.html`

```
    {{ template "authflowv2/__alert_message.html"
      (dict
        "Type" "error"
        "Classname" "mt-4"
        "Message" (include "authflowv2/__error.html" .)
      )
    }}
```

Reply:
Right, Lemme change that 🙏 

Is there doc for `how to setup oauth locally`? namely wechat and google 

Otherwise will just follow official authgear doc like this one https://docs.authgear.com/how-to-guide/how-to-setup-sso-integrations/social-login-providers/wechat

Reply:
You cannot set up Wechat easily because it requires approval from Wechat to create an application.

Reply:
For google, you just follow the official doc.

Reply:
> You cannot set up Wechat easily

Oh.. thanks for the heads up. Lemme still apply for future.

 I guess I just blindly change the error and rely on you to help me test it this time 🙇